fullscreenApp.css

```diff
@@ -0,0 +1,8 @@
+html {
+  overflow: hidden;
+}
+
+html, body, #app {
```

Comment:
`#app` is clearly opinionated, but it's the consolidated standard in buildo's next-gen projects (revenge apps)